### ISSUE TITLE
Debugger steps into instrumented version of non-spec code

### ISSUE THREAD

**Issue by VilleSalonen** [monounity/karma-typescript#15]
**tl;dr:** Chrome debugger uses instrumented version of non-spec code instead of original TypeScript code. How can I disable instrumentation during debugging of tests?

I created an example of the problem in my own branch: https://github.com/VilleSalonen/karma-typescript/tree/broken-example

Preliminary steps:
```
git clone https://github.com/VilleSalonen/karma-typescript.git
cd karma-typescript
git checkout broken-example
cd example-project
npm install
npm test
```

Open Debug view of Karma, navigate to hello.component.ts and set breakpoints on lines 14 and 15:

![image](https://cloud.githubusercontent.com/assets/1070813/19645196/3d7727fa-99fc-11e6-96ea-a87398315f9e.png)

Refresh the page and step into log1 function on line 14. log1 is imported from non-spec file example1.ts so it has been instrumented. Chrome steps into the instrumented version of the code:

![image](https://cloud.githubusercontent.com/assets/1070813/19645228/64cd9816-99fc-11e6-8a6c-40ea2d7491a1.png)

However if you step into log2 function on line 15, you get the original TypeScript code. log2 is imported from example2.spec.ts so it has not been instrumented:

![image](https://cloud.githubusercontent.com/assets/1070813/19645277/a5824b90-99fc-11e6-8eb9-02c775c50e60.png)

**Comment by VilleSalonen** [comment]
I solved the problem quick and dirty with following ShellJS script but that is awful hack and I'd like to get rid of that as soon as possible. I also added a pull request for disableCodeCoverageInstrumentation flag if there are indeed no such flags or other mechanisms before.

```
/*
This is absolutely terrible hack to fix debugging tests with karma-typescript.

The problem is that non-spec files get instrumented with Istanbul automatically
and if you try to step into code imported from non-spec file, you don't get
the original TypeScript code: you get the really ugly instrumented code.

If you know of a better way to disable Istanbul instrumentation of non-spec
files, please use that instead of this monstrosity.

Here's a Github issue related to the problem:
https://github.com/monounity/karma-typescript/issues/15
*/

if (process.argv[3] == "ENABLE") {
    sed('-i', /true \/\*0c067128-a85e-4aab-a107-05c0bf20f833\*\//, "specFileRegex.test(file.originalPath)", 'node_modules/karma-typescript/lib/preprocessor.js');
}

if (process.argv[3] == "DISABLE") {
    sed('-i', /specFileRegex\.test\(file\.originalPath\)/, 'true \/\*0c067128-a85e-4aab-a107-05c0bf20f833\*\/', 'node_modules/karma-typescript/lib/preprocessor.js');
}
```
